packages/components/src/components/editor/browser/browser.stories.tsx

```diff
@@ -121,29 +138,33 @@ export const Default: Story = {
   render: () => <DefaultBrowser />
 };
 
+const DialogBrowserInput = ({ value, onChange }: { value: string; onChange: (change: string) => void }) => {
+  const [open, setOpen] = useState(false);
+  return (
+    <Dialog open={open} onOpenChange={setOpen}>
+      <InputGroup style={{ flex: '1' }}>
+        <Input value={value} onChange={e => onChange(e.target.value)} />
+        <DialogTrigger asChild>
+          <Button icon={IvyIcons.ListSearch} aria-label='Browser' />
+        </DialogTrigger>
+      </InputGroup>
+      <DialogContent style={{ height: '80vh' }}>
+        <DefaultBrowser
```

Comment:
Isn't it possible to open in this browser also a condition builder? 🙈

Reply:
Oh yes, that's right 😄, I can add a flag that the condition builder is not integrated for this case.